src/test/scripts/org/reaktivity/k3po/nukleus/ext/simplex/server.sent.throttle.initial.only.update.none/client.rpt

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+connect "nukleus://server/streams/client"
+        option nukleus:route 1234L
+        option nukleus:transmission "simplex"
+        option nukleus:window 8192
```

Comment:
remove this line (not relevant for simplex on client side)

Reply:
If I don't add this line, I get the following error: 
```
org.kaazing.k3po.junit.rules.SpecificationException: Internal error:nukleus window option missing
	at org.kaazing.k3po.junit.rules.ScriptRunner.call(ScriptRunner.java:135)
	at org.kaazing.k3po.junit.rules.ScriptRunner.call(ScriptRunner.java:42)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)


```

Reply:
OK. I have filed https://github.com/reaktivity/k3po-nukleus-ext.java/issues/23 for this bug.